book.toml

```diff
@@ -13,14 +13,14 @@ extra-watch-dirs = ["po"]
 
 [preprocessor.links]
 renderers = ["html"]
-before = ["gettext"]
+
+[preprocessor.gettext]
 
 [preprocessor.index]
 renderers = ["html"]
 
 [preprocessor.svgbob]
 renderers = ["html"]
-after = ["gettext"]
```

Comment:
Ah, are you able to remove this because the preprocessor order is determined by the order in `book.toml`? That's nice!

Reply:
Yup, you're right.